Lib/Time/TimeUtil.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace CakePHPUtil\Lib\Time;
+
+/**
+ * Class TimeHelper
+ * Put time manipulation methods here.
+ */
+class TimeUtil {
+
+	/**
+	 * @param $hours
+	 *
+	 * @return string in the form [-]hh[:mm[:ss]]
+	 */
+	public function hoursToDecimal($hours) {
+		$hours = trim($hours);
+		if (is_null($hours)) {
+			return '';
+		}
+		$sign = 1;
+		if (substr($hours, 0, 1) === '-') {
+			$sign = -1;
+			$hours = substr($hours, 1);
+		}
+		$parts = explode(':', $hours);
+		foreach ([
+			'hours',
+			'minutes',
+			'seconds'
+		] as $index => $name) {
+			${$name} = $parts[$index] ?? 0;
+		}
+
+		return $sign * ($hours + ($minutes / 60) + (($seconds ?? 0) / 3600));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$minutes" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS1481)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$seconds" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS1481)